iomanagers/redis_manager.py

```diff
@@ -0,0 +1,19 @@
+"""
+    Redis manager is responsible for managing Redis connections and various IO operations
+    from the autocomplete server to one Redis server.
+"""
+import redis
+
+
+class RedisManager:
+    def __init__(self, redis_host: str, redis_port: str, db_idx: int):
```

Comment:
- What does the db_idx signify here ?

Reply:
It's database index, in case we have more than one database.